JapaneseFilter/sections/specific.txt

```diff
@@ -1998,10 +1995,23 @@ intensive911.com##div[id^="adx_"]
 clicccar.com##.ad-management
 quick-timez.com##.k1y-inline-ad
 ferret-plus.com##div[class*="-ad-banner"]
-itmedia.co.jp###ITN
+/ad.js$domain=itmedia.co.jp
+||itmedia.co.jp^*/file_bit_article_bottom_ad.html
+||itmedia.co.jp/css/sa.js
+||itmedia.co.jp/css/sa_ini.js
 itmedia.co.jp###ATF
+itmedia.co.jp###ITN
+itmedia.co.jp###cmsBody > div.inner > div > div[style="text-align:center;margin:40px 0;"]
 itmedia.co.jp###cmsBodyMain > div:not([class]) > div[style="text-align:center;margin-bottom:20px;"]:only-child
 itmedia.co.jp##div[style^="font-family:Gill Sans;"][style*="font-size:12px;"][style*="line-height:12px;"]
+itmedia.co.jp##div[id^="ITR"]
```

Comment:
Better `###ITR` + `itmedia.co.jp###globalHeader`

Reply:
`###ITR`,`###ITR2` and `###ITRN` are there.
Should I write the rule separately?

Reply:
Hm, I missed ITR2. In that case maybe `##div[id^="ITR"]` is maybe okay. Butno need for `###ITRN` as covered by `###globalHeader`.

Reply:
I see.